Modules/TRD/src/TrackletPerTriggerCheck.cxx

```diff
@@ -60,6 +62,14 @@ void TrackletPerTriggerCheck::configure() {
     ILOG(Debug,Support)<<"configure() : using default Upperthreshold"<<DesiredMeanRegion.second<<ENDM;
   }
 
+  if(auto param =mCustomParameters.find("StatThreshold");param != mCustomParameters.end()){
+    StatThreshold=stod(param->second);
+    ILOG(Debug,Support)<<"configure() : using json StatThreshold"<<StatThreshold<<ENDM;
+  }else{
+    StatThreshold=0;
```

Comment:
I'd put something like 1000 as default. Zero does not make sense and 1000 is something which should be reached very quickly

Reply:
Ok, I will put 1000.